t/16-utils-vcs-provider.t

```diff
@@ -7,7 +7,7 @@ use Test::Most;
 use FindBin;
 use lib "$FindBin::Bin/lib", "$FindBin::Bin/../external/os-autoinst-common/lib";
 require OpenQA::Test::Database;
-use OpenQA::Test::TimeLimit '5';
+use OpenQA::Test::TimeLimit '25';
```

Comment:
As you bumped the timeout without any other changes it might be worth checking if/why the test is slow for you. Generally it should work unmodified